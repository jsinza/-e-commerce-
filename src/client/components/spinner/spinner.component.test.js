import React from 'react';
import 'babel-polyfill';
import { mount } from 'enzyme';
import Spinner from './spinner.component';

describe('Spinner Component', () => {

  const wrapper = mount(<Spinner />);;

  it('should render BreadCrumb component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
