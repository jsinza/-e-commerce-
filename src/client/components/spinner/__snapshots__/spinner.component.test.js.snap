// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component should render BreadCrumb component 1`] = `ReactWrapper {}`;
